test-suite/src/test/scala/org/scalajs/testsuite/jsinterop/ExportsTest.scala

```diff
@@ -966,6 +966,56 @@ object ExportsTest extends JasmineTest {
       expect(jsPackage.ExportObjSetterNamed_=().x).toBe(1)
     }
 
+    it("should export js.Function values of new js.Object{...} - #1899") {
```

Comment:
The title of the test has virtually nothing to do with the test. I suggest:

    should expose public members of new js.Object {...} despite SI-9487 - #1899